__tests__/app.test.js

```diff
@@ -72,3 +72,17 @@ describe('.all(*) - handles any endpoint that does not exist', () =>{
             })
         });
     })
+
+describe('/api/articles', () => {
+    test('should return an array of article objects with the correct shape.', () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({body})=>{
+            expect(body.articles.length).toBe(5)
+            const articles = body.articles
+            expect(articles).toBeSorted({descending: true})
```

Comment:
We should also have expect statements here which show the structure of the articles is correct. What keys are we expecting and what data with them?